main.go

```diff
@@ -0,0 +1,42 @@
+package main
+
+import (
+	"errors"
+	"fmt"
+	"os"
+	"regexp"
+
+	"github.com/weaveworks/release-note-action/action"
+)
+
+func main() {
+	a, err := action.New(releaseNotAction)
+	if err != nil {
+		fmt.Fprintln(os.Stdout, "failed to create action: %w", err)
+		os.Exit(1)
+	}
+
+	if err := a.Do(); err != nil {
+		fmt.Printf("action failed: %s\n", err.Error())
+		os.Exit(1)
+	}
+}
+
+func releaseNotAction(ctx *action.Context) error {
```

Comment:
typo `releaseNoteAction` ?

Reply:
The code has been refactored and this function has been removed.